packages/daemon/templates/node/predict.js

```diff
@@ -29,9 +29,9 @@ async function predict(data) {
   if (!model) {
     model = await modelDefine;
   }
-  const sample = { data, label: null };
+  let sample = { data, label: null };
   if (typeof dataProcessModule === 'function') {
-    await dataProcessModule(sample, {}, JSON.parse(pipeline.dataProcessParams));
+    sample = await dataProcessModule(sample, {}, JSON.parse(pipeline.dataProcessParams));
```

Comment:
Dos the __data-process__ plugin return `Sample` now?

Reply:
Exactly, data-process and global-data-process will not modify original data.

Reply:
See https://github.com/alibaba/pipcook/blob/master/docs/spec/plugin.md#plugin-category, this needs an update IMO.

Reply:
Done ~~